packages/component-library/src/components/plmg-slider/plmg-slider.tsx

```diff
@@ -83,13 +83,12 @@ export class Slider {
   @Prop() rangeValues: string;
   @Watch('rangeValues')
   validateRangeValues(newValue: string) {
-    if (
+    if (newValue)
       !Array.isArray(this.stringToNumberArray(newValue)) ||
-      this.stringToNumberArray(newValue).length < 2
-    )
-      throw new Error(
-        'rangeValues must be a comma separated list with at least two items'
-      );
+        this.stringToNumberArray(newValue).length < 2;
```

Comment:
Changed. Now it will just throw an error if the array is
- not a string
- an empty string
- an array with less than two items
- if one of the items in the array is not a number